src/colorizer/workers/worker.ts

```diff
@@ -34,10 +34,10 @@ async function getMotionDeltas(
   trackIds: Uint32Array,
   times: Uint32Array,
   centroids: Uint16Array,
-  config: VectorConfig
+  timeIntervals: number
```

Comment:
The only value in the config that was being used was the time interval, so I ended up changing this argument to be the time interval only.